README.md

```diff
@@ -1,2 +1,43 @@
+
+The UI in Adobe Experience Platform that provides Platform Launch capabilities has now been renamed as the Data Collection UI. When a concept or capability is described as being "in" Launch, the author is likely referring to the Data Collection UI as it was previously known as the Launch UI. 
+
+Despite "Adobe Experience Platform Data Collection" appearing in the UI, you should avoid having "Adobe Experience Platform" and "data collection" next to each other since it is not a branded service. Data collection is only capitalized in reference to the UI or when it is directly referred to as category, although that is rare. An example of this is provided in the table below.
+
+| Original Term  |Updated Term | Explanation  |Previous Example Sentence  | Current Branded Use |
+|---|---|---|---|---|
+| **When configured in Adobe Experience Platform Launch, ...**  | When configured in the Data Collection UI... | When referring to something "in" Launch, it is assumed that the author is referring to the UI.  | **When configured in Adobe Experience Platform Launch**, this extension sends AEM Asset Impression and Click metrics to Adobe Analytics.  | **When configured in the Data Collection UI**, this extension sends AEM Asset Impression and Click metrics to Adobe Analytics.  |
+
+The table below offers more comprehensive guidance on the varying uses of the original terms and how to apply the branding updates in a given situation. As any given term can be used in a variety of contexts, the table offers guidance on the nuance of the branding changes, including how they are applied and an explanation as to why.
+
+| Original Term  |Updated Term | Explanation  |Previous Example Sentence  | Current Branded Use |
+|---|---|---|---|---|
+| **Platform Launch**  | Platform  | The reference to Launch is not always necessary if the sentence was referring to the service provided. In this case Launch can be omitted as tags is a set of capabilities and not a service. Here, Platform provides the functionality.  | **Platform Launch** passes `settings` into the rule’s condition-type library modules where conditions are then evaluated. | **Platform** passes `settings` into the rule’s condition-type library modules where conditions are then evaluated. |
+| **Platform Launch** | {ommitted} | If context has already been established, naming the capability in the sentence may be obsolete and can be removed entirely. | If the extensions, tags, and systems you are going to deploy through **Platform Launch** are very similar across your sites or applications, you might want to include them in the same property. | If the extensions, tags, and systems you are going to deploy are very similar across your sites or applications, you might want to include them in the same property. |
```

Comment:
Remove this row and add a separate section on when to omit "Launch". Having "{ommitted}" as an "Updated term" is not intuitive and is going to confuse people. Don't bury this in misc info.